cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -128,13 +127,12 @@ public MapReduceManager startMapReduce(final String jobName) {
 
   @Override
   public MapReduceManager startMapReduce(final String jobName, Map<String, String> arguments) {
-    return getMapReduceManager(jobName, arguments, ProgramType.MAPREDUCE);
+    final Id.Program jobId = startProgram(jobName, arguments, ProgramType.MAPREDUCE);
```

Comment:
rename to programId, since there is a concerted effort to not use job anywhere in CDAP.